source/braille.py

```diff
@@ -720,6 +720,9 @@ def getControlFieldBraille(info, field, ancestors, reportStart, formatConfig):
 		level = field.get("level")
 		if level:
 			props["positionInfo"] = {"level": level}
+		if value:
+			props['value']=value

```

Comment:
The `value` key is already set when the `props` dict is created. One of these is redundant.